.github/workflows/containers.yml

```diff
@@ -15,7 +15,8 @@ jobs:
 
       - name: Build containers
         run: |
-          docker build . --tag topo-imagery --label "github_run_id=${GITHUB_RUN_ID}"
+          docker build -f Dockerfile.imagery . --tag topo-imagery --label "github_run_id=${GITHUB_RUN_ID}"
```

Comment:
create two different docker images

Reply:
Probably review what to do for the `--label` here

Reply:
Haven't look at publishing one container or the other depending of what is merged. Don't know if it's easy or not